config/install/editor.editor.full_html.yml

```diff
@@ -57,10 +57,6 @@ settings:
           items:
             - WebsparkTable
   plugins:
-    stylescombo:
-      styles: "mark|Highlight text copy\r\nspan.highlight-gold|Gold Highlight\r\nspan.highlight-black|Black Highlight\r\nspan.highlight-white|White Highlight\r\nspan.btn.btn-gold|Gold Button\r\nspan.btn.btn-maroon|Maroon Button\r\nspan.btn.btn-gray|Light Button\r\nspan.btn.btn-dark|Dark Button\r\ndiv.span.copy-divider.gold| Copy Divider Gold\r\ndiv.span.copy-divider.dark| Copy Divider Black\r\ndiv.span.copy-divider.maroon| Copy Divider Maroon"
-    language:
-      language_list: un
```

Comment:
Is this removed intentionally?

Reply:
It was because they weren't in my current, local active configs (or start point configs), but I rechecked the current webspark-profile-webspark YMLs and the settings were still there. This also applied to the config/install/editor.editor.basic_html.yml file so I restored those as well (see the latest commit).

I'm not sure how those were removed locally from my active configs because I was doing this work from a clean install.